nvflare/app_opt/xgboost/histogram_based_v2/defs.py

```diff
@@ -100,7 +100,9 @@ class Constant:
     EVENT_AFTER_ALL_GATHER_V = "xgb.after_all_gather_v"
 
     HEADER_KEY_ENCRYPTED_DATA = "xgb.encrypted_data"
+    HEADER_KEY_HORIZONTAL = "xgb.horizontal"
     HEADER_KEY_ORIGINAL_BUF_SIZE = "xgb.original_buf_size"
+    HEADER_KEY_IN_AGGR = "xgb.in_aggr"
```

Comment:
If I understand correctly, shall we rename the key names "encrypted_data", "horizontal", and "in_aggr" to "KEY_SYMMETRIC", "KEY_ASYMMETRIC_PRIVATE" and "KEY_ASYMMETRIC_PUBLIC" for easier understanding?